apps/indexer/lib/indexer/block/fetcher.ex

```diff
@@ -211,16 +224,16 @@ defmodule Indexer.Block.Fetcher do
          } = CeloAccounts.parse(logs, oracle_address),
          market_history =
            exchange_rates
-           |> Enum.filter(fn el -> el.token == stable_token end)
+           |> Enum.filter(fn el -> el.token == stable_token_usd end)
            |> Enum.filter(fn el -> el.rate > 0 end)
            |> Enum.map(fn %{rate: rate, stamp: time} ->
              inv_rate = Decimal.from_float(1 / rate)
              date = DateTime.to_date(DateTime.from_unix!(time))
              %{opening_price: inv_rate, closing_price: inv_rate, date: date}
            end),
          exchange_rates =
-           (if Enum.count(exchange_rates) > 0 and gold_token != nil do
-              [%{token: gold_token, rate: 1.0} | exchange_rates]
+           (if Enum.count(exchange_rates) > 0 and celo_token != nil do
+              [%{token: celo_token, rate: 1.0} | exchange_rates]
```

Comment:
Maybe we should remove this and just get the exchange rate from Coingecko or somewhere...